MacDown/Code/Document/MPDocument.m

```diff
@@ -1363,6 +1345,32 @@ - (IBAction)render:(id)sender
 
 #pragma mark - Private
 
+- (void)toggleSplitterCollapseWithVisibility:(BOOL)isVisible
+                                 targetRatio:(CGFloat)targetRatio
```

Comment:
The problem is that the ternary operator is `0.0 : 1.0` for the preview pane and `1.0 : 0.0` for the editor pane. After the method `toggleSplitterCollapseWithVisibility` is called, the information of which panel is the one we want is "lost". I suppose we could change the method to take the panel as a parameter instead of the ratio, but it would still need two.